pkg/investigations/chgm/chgm.go

```diff
@@ -30,6 +31,11 @@ var chgmLimitedSupport = ocm.LimitedSupportReason{
 	Details: "Your cluster is no longer checking in with Red Hat OpenShift Cluster Manager. Possible causes include stopped instances or a networking misconfiguration. If you have stopped the cluster instances, please start them again - stopping instances is not supported. If you intended to terminate this cluster then please delete the cluster in the Red Hat console",
 }
 
+// ClusterDeploymentSupportExceptionLabel is the label indicating the cluster is under limited support
+// and the PagerDuty service should be enabled even if the cluster is in limited support
+// TODO:() Centrailize or Remove this varaiable once https://issues.redhat.com/browse/XCMSTRAT-427 is completed
+var ClusterDeploymentSupportExceptionLabel = "hive.openshift.com/support-exception"
```

Comment:
```suggestion
var ClusterDeploymentSupportExceptionLabel = "ext-managed.openshift.io/support-exception"
```

@rafael-azevedo I updated the label